js/plugins/helperLines.js

```diff
@@ -2,11 +2,11 @@
  * Displays the x-coordinate i.e the label and y-coordiante i.e the day
  * when clicked on a timeline element in advanced view.
  */
-function initHelperLines(enableHelperLines, zoomFactor, advancedView, divId, formatTime, daysToTimeObject) {
+function initHelperLines(options) {
   //enable helper lines only if no zoom and in advanced view
   d3.selectAll("[id^='timelineItem']").on("click", function(x) {
-    if (enableHelperLines && zoomFactor === 1 && advancedView) {
-      var helperLineGroup = d3.select(divId + " svg").append("g").attr("class", "helper-line-group");
+    if (options.zoomFactor === 1 && options.timelineBool.advancedView) {
```

Comment:
This shows some tight integration with the zoom plugin. You could perhaps check whether that plugin is enabled, run if not, otherwise only run if the `zoomFactor === 1`.